ext/http/http_next.rs

```diff
@@ -345,16 +345,24 @@ fn is_request_compressible(headers: &HeaderMap) -> Compression {
   if accept_encoding == "gzip" {
     return Compression::GZip;
   }
+  if accept_encoding == "br" {
+    return Compression::Brotli;
+  }
```

Comment:
Let's change this and the lines above to a `match ...` -- and if `"gzip, deflate, br"` is passed, it should choose Brotli.